cmd/perf-tests.go

```diff
@@ -61,6 +62,14 @@ func selfSpeedtest(ctx context.Context, size, concurrent int, duration time.Dura
 	var totalBytesWritten uint64
 	var totalBytesRead uint64
 
+	client, err := minio.NewCore(globalLocalNodeName, &minio.Options{
+		Creds:  credentials.NewStaticV4(globalActiveCred.AccessKey, globalActiveCred.SecretKey, ""),
+		Secure: globalIsTLS,
+	})
+	if err != nil {
+		return SpeedtestResult{}, err
+	}
```

Comment:
you might also want to use the globalProxyTransport that trust all the certs etc. 

```
        opts := &minio.Options{
                Creds:     creds,
                Secure:    globalIsTLS,
                Transport: globalProxyTransport,
        }
```